modules/client/examples/01-client/13-prepare-installation.ts

```diff
@@ -5,18 +5,18 @@ title: PrepareInstallation
 ### Prepare installation of a plugin
 */
 
+import { Client } from "@aragon/sdk-client";
 import {
-  Client,
   PrepareInstallationParams,
   PrepareInstallationStep,
-} from "@aragon/sdk-client";
+} from "@aragon/sdk-client-common";
 import { context } from "../index";
 
 // Instantiate the general purpose client from the Aragon OSx SDK context.
 const client: Client = new Client(context);
 
 // Obtained from the plugin metadata
-const installationAbi = [ ... ];
+const installationAbi = [/* ... */];
 
 const prepareInstallationParams: PrepareInstallationParams = {
```

Comment:
When I ran this locally, the type `PrepareInstallationParams` did not exist. Just adding this make sure it does when you're running the example

Reply:
Amazing! Thank you